image/templates/helm/shared/config-templates/scanner-v4-db/postgresql.conf.default

```diff
@@ -30,7 +30,7 @@ ssl_key_file = '/run/secrets/stackrox.io/certs/server.key'
 
 # - Memory -
 
-shared_buffers = 128MB
+shared_buffers = 250MB
```

Comment:
Check out https://www.postgresql.org/docs/current/runtime-config-resource.html for guidance for setting this value